workflow/Snakefile

```diff
@@ -19,7 +19,7 @@ SBS_SAMPLE_FPS = list(sbs_samples_df["sample_fp"].unique())
 if len(SBS_SAMPLE_FPS) > 0:
     SBS_WELLS = list(sbs_samples_df["well"].unique())
     SBS_TILES = list(sbs_samples_df["tile"].unique())
-    SBS_CYCLES = list(sbs_samples_df["cycle"].unique())
+    SBS_CYCLES = sorted(list(sbs_samples_df["cycle"].unique()))
```

Comment:
What was the logic behind this?

We should be sorting by all metadata in `create_samples_df`. Maybe there is a bug in this function?